modules/social_features/social_user/social_user.module

```diff
@@ -266,27 +266,6 @@ function _social_user_search_content_pre_render(array $build) {
   return $build;
 }
 
-/**
- * Implements hook_user_format_name_alter().
- */
-function social_user_user_format_name_alter(&$name, $account) {
```

Comment:
@jochemvn don't we still need this? When the Social Profile Fields module is turned off, is anything still taking care of this?

Reply:
@bramtenhove yes we indeed still need it... But since I felt, that the social_profile module was a better place than social_user module, I've taken the liberty to move it there (it's not removed 😉)

Reply:
@bramtenhove see: https://github.com/goalgorilla/open_social/pull/731/commits/b6565b88b907f7faec761ac1b636ef14310381b1

Reply:
@jochemvn Ok cool! I overlooked the fact that you moved it 😅